### PR TITLE
fix `Constant` Activity in examples to `Const`

### PR DESCRIPTION
**Pull request by kiranshila**

### CODE REVIEW

**Comment by activexray** [comment]
agh my formatter seems to have mucked with a few things

**Comment by activexray** [comment]
Also, not sure where to ask about this - but in the examples, you have the likes of
```rust
#[autodiff(backprop, Reverse, Const, Duplicated, Active)]
fn training_loss(images: &[f32], weights: &[f32]) -> f32 {
  let loss = do_some_math(images, weights);
  loss
}
```
generating code like
```rust
fn backprop(images: &[f32], weights: &[f32], dweights: &mut [f32]) -> f32
```
but this isn't quite correct, right? There's an extra f32 seed parameter, if I'm not mistaken.

**Comment by ZuseZ4** [comment]
Thanks!
Yes, some parts were unfortunately written before we had CI working, so we forgot to update them when changing autodiff. I'm currently in the process of upstreaming all tests into rust/tests, and in the future let the Rust CI make sure that we don't regress them again. I also have more real-world examples these days of differentiating e.g. Karpathy's llm.c (a rust version of it), as well as Microsoft's ADBench implementation, so I intend to completely replace the outdated examples at some point.
